sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py

```diff
@@ -382,7 +388,6 @@ def send_to_connection(  # pylint: disable=inconsistent-return-statements
         _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
         _params = kwargs.pop("params", {}) or {}
 
-        content_type = kwargs.pop("content_type", _headers.pop("Content-Type", "application/json"))  # type: str
```

Comment:
If you don't set default value for content-type, existing users may be broken. Is it expected?